src/block/Door.php

```diff
@@ -176,4 +177,15 @@ public function getAffectedBlocks() : array{
 	private function canBeSupportedAt(Block $block) : bool{
 		return $block->getAdjacentSupportType(Facing::DOWN)->hasEdgeSupport();
 	}
+
+	public function onWindChargeInteraction(WindCharge $windCharge) : void{
+		if($this->getTypeId() == BlockTypeIds::IRON_DOOR) {
+			return;
+		}
+
+		$this->open = !$this->open;
+		$world = $this->position->getWorld();
+		$world->setBlock($this->position, $this);
+		$world->addSound($this->position, new DoorSound());
```

Comment:
need to used the same logic as interact into one single function to be sure to update the top door block